pkg/identity/keystone/token_getter.go

```diff
@@ -64,7 +68,28 @@ func GetToken(options Options) (*tokens3.Token, error) {
 		}
 		tlsConfig.Certificates = []tls.Certificate{cert}
 		tlsConfig.BuildNameToCertificate()
+		setTransport = true
+	}
+
+	if options.ClientCAPath != "" {
+		certBytes, err := ioutil.ReadFile(options.ClientCAPath)
+		if err != nil {
+			msg := fmt.Errorf("failed: Cannot read CA file: %v", err)
+			return token, msg
+		}
+
+		certpool, err := x509.SystemCertPool()
```

Comment:
could we use kk lib instead here "k8s.io/client-go/util/cert" which makes consistent with other modules ? and directly use cert.NewPool(options.ClientCAPath) which handles the reading file etc?

Reply:
@ramineni Thanks for the pointer. I like that much better and consistency is best!